source/developers/add_new_platform.markdown

```diff
@@ -61,3 +61,29 @@ The Discovery component is capable of setting up your components before firing t
 <p class='note warning'>
 This option is currently limited to built-in components.
 </p>
+
+### {% linkable_title Creating Entities %}
+
+Home Assistant will call a function with the following signature to initialize
```

Comment:
I changed all references to component to references to platform. I'm not entirely sure if this is more clear now. The more I think about it, the more I just confuse myself.